spec/01-unit/26-tracing/01-tracer_pdk_spec.lua

```diff
@@ -1,6 +1,18 @@
 require "spec.helpers" -- initializes 'kong' global for tracer
 local match = require("luassert.match")
 
+local utils = require "kong.tools.utils"
+local SAMPLING_BYTE = 8
+local rand_bytes = utils.get_rand_bytes
+local TEST_COUNT = 10000
+-- we can only ensure a sampling precision of 0.01
+local SAMPLING_PRECISION = 0.01
```

Comment:
Running the test a few times I got some failures like:
`sampling rate is not correct: 0.4903 expected: 0.5007118` - we should probably adjust this precision

Reply:
Thanks for testing this. Changed to 0.02.